changelog.txt

```diff
@@ -1,3 +1,17 @@
+4.24.0
+============
+
+New Features:
+**Performance improvement by reducing the number of calls to GetAllAccessTokens()**. See issue [#2204](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2204) for details.
+**MSAL .NET now sends the whether the token cache is serialized to telemetry**. See issue [#2185](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2185) for details.
```

Comment:
might re-read this...not sure what you're trying to say ;)

Reply:
I don't think we should advertise this. It's not a feature from a customer perspective.

Reply:
we have a new section, which is "Experimental Features" it was @henrik-me 's idea for the last release. i forgot about it. that would go in that section.

Reply:
This is not specific to regional. This is for all the requests. But agree with @bgavrilMS this is for internal tracking. Can remove it from here.